packages/plugins/Fullscreen/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Feature: Hide obstructive tooltip on small devices.
 - Fix: Documentation error regarding plugin state.
+- Fix: Since iPhones do not support fullscreen mode, the fullscreen button is hidden on such devices.
```

Comment:
```suggestion
- Fix: Since iPhones do not support fullscreen mode, the fullscreen button is hidden on such devices. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Fullscreen_API#browser_compatibility) for more information.
```